docset/winserver2012-ps/dism/Add-AppxProvisionedPackage.md

```diff
@@ -138,10 +138,10 @@ Accept wildcard characters: False
 Specifies the maximum output level shown in the logs.
 The default log level is 3.
 The accepted values are as follows:
-1 = Errors only
-2 = Errors and warnings
-3 = Errors, warnings, and information
-4 = All of the information listed previously, plus debug output
+- 1 = Errors only
+- 2 = Errors and warnings
+- 3 = Errors, warnings, and information
+- 4 = All of the information listed previously, plus debug output
```

Comment:
```suggestion
- Errors
- Warnings
- WarningsInfo (default)
```

Reply:
I think it should be the same as https://github.com/MicrosoftDocs/windows-powershell-docs/pull/2139.

Could you test it in 2012?

Reply:
Seems like it. Could be a broader issues on the` -LogLevel `parameter in the module... @e0i This could probably be investigated.

Confirmed on 2012, 2012 R2, 2016 and Win10.